app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/model/QuickLink.kt

```diff
@@ -1,26 +1,18 @@
 package com.hedvig.android.feature.help.center.model
 
-import androidx.annotation.StringRes
 import com.hedvig.android.navigation.core.AppDestination
-import hedvig.resources.R
-import kotlinx.collections.immutable.persistentListOf
 
-internal enum class QuickLink(
-  val destination: AppDestination,
-  @StringRes val titleRes: Int,
-) {
-  ChangeBank(AppDestination.ConnectPayment, R.string.HC_QUICK_ACTIONS_CHANGE_BANK),
-  UpdateAddress(AppDestination.ChangeAddress, R.string.HC_QUICK_ACTIONS_UPDATE_ADDRESS),
-  EditCoInsured(
-    AppDestination.TopLevelDestination.Insurance,
-    R.string.HC_QUICK_ACTIONS_EDIT_COINSURED,
-  ), // TODO go to specific contract id and start add or edit co-insured
-  TravelCertificate(AppDestination.GenerateTravelCertificate, R.string.HC_QUICK_ACTIONS_TRAVEL_CERTIFICATE),
-}
+sealed interface QuickAction {
+  val titleRes: Int
+
+  data class MultiSelectQuickLink(
+    override val titleRes: Int,
+    val links: List<QuickLink>,
+  ) : QuickAction
 
-internal val quickLinks = persistentListOf(
-  QuickLink.ChangeBank,
-  QuickLink.UpdateAddress,
-  // QuickLink.EditCoInsured,
-  QuickLink.TravelCertificate,
-)
+  data class QuickLink(
+    override val titleRes: Int,
+    val displayName: String,
+    val destination: AppDestination,
```

Comment:
Great work making this work like this, great usage of how we do nav.
I think this could also just be Destination (the supertype of App Destination, and all destinations actually), so we're not confined just to the top level ones perhaps?

Reply:
Yes good idea